md/optimize-user-tasklist.md

```diff
@@ -50,11 +50,13 @@ To make sure the process gets the information it needs to start a new instance,
 3. Click **Next**
 4. Unselect **requesterName** and **status** as the process doesn't need this information from the form to instantiate  
    The default values of business data attributes mapped to contract inputs are automatically set thanks to a script generated along with the contract inputs themselves.  
-5. Click **Preview** to view this script, and then **Finish**, **OK** and **OK**
+5. Click **Finish**, carefully read the warnings, and press **OK** and **OK**.
 6. In the **Execution** pane > **Contract** tab, a complex contract input is created, mapped to the selected attributes of the **leaveRequest** business variable, as shown here:
 
 ![Contract](images/contract_for_tasklist.png)<!--{.img-responsive .img-thumbnail}-->
 
+In actual BPM projects, we recommend you also add a description to each contract input. It will be used as input field caption for end-users in the auto-generated form, if you decide to use such forms up to the User Acceptance Test phase of your project.  
```

Comment:
i can add one "we recommend you to also add". The sentence is correct without, but yeah, it's easier to understand with it.